scripts/save_org_product_data_daily.pl

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use ProductOpener::PerlStandards;
+
+use CGI::Carp qw(fatalsToBrowser);
+
+use ProductOpener::Config qw/:all/;
+use ProductOpener::Data qw/:all/;
+use ProductOpener::Paths qw/%BASE_DIRS ensure_dir_created/;
+use ProductOpener::Users qw/$Owner_id/;
+use ProductOpener::Orgs qw/retrieve_org/;
+use Storable qw(store);
+
+# This script is run daily to gather organisation data
+# such as number of products, number of products with errors etc,
+
+ensure_dir_created($BASE_DIRS{ORGS});
+
+my $products_collection = get_products_collection();
+my $orgs_collection = get_orgs_collection();
+
+sub get_org_data ($org_id) {
+	my $org_data = $products_collection->aggregate(
+		[
+			{'$match' => {'owner' => "org-" . $org_id}},
+			{
+				'$group' => {
+					'_id' => '$owner',
+					'number_of_products' => {'$sum' => 1},
+					'number_of_data_quality_errors' => {
+						'$sum' => {'$size' => '$data_quality_errors_tags'}
+					},
+					'number_of_data_quality_warnings' => {
+						'$sum' => {'$size' => '$data_quality_warnings_tags'}
+					},
+					'number_of_products_without_nutriscore' => {
+						'$sum' => {
+							'$cond' => [{'$in' => ['en:nutriscore-not-computed', '$misc_tags']}, 1, 0]
+						}
+					},
+					'opportunities_to_improve_nutriscore' => {
+						'$sum' => {
+							'$cond' => [{'$in' => ['possible-improvements', '$misc_tags']}, 1, 0]
+						}
+					},
+					'products_with_changes_since_last_export' => {
+						'$sum' => {
+							'$cond' => [{'$regexMatch' => {input => '$states', regex => 'en:to-be-exported'}}, 1, 0]
+						}
+					},
+					'date_of_last_update' => {'$max' => '$last_modified_t'}
+				}
+			}
+		]
+	)->next;
+
+	my $number_of_products = $org_data->{number_of_products} // 0;
```

Comment:
In another PR, we will need to make another query for number_of_products, but on the public platform, in order to get the number of products from the pro platform that have been exported to the pro platform.